src/utils/graph/getGraphMarkets/markets.formatter.ts

```diff
@@ -1,17 +1,19 @@
 import { BigNumber } from "ethers";
-import { GraphMarketConfiguration } from "./markets.types";
+import { GraphMarketConfiguration, MarketMinimal } from "./markets.types";
 import { CompoundMath } from "@morpho-labs/ethers-utils/lib/maths";
 
-export const graphToMarketConfig = (graphMarket: GraphMarketConfiguration) => ({
+export const graphToMarketConfig = (graphMarket: GraphMarketConfiguration): MarketMinimal => ({
   address: graphMarket.address,
   price: BigNumber.from(graphMarket.reserveData.usd),
-  totalSupply: CompoundMath.mul(graphMarket.metrics.totalSupplyOnPool, graphMarket.reserveData.supplyPoolIndex),
-  totalBorrow: BigNumber.from(graphMarket.metrics.totalBorrowOnPool),
+  totalPoolSupplyUSD: CompoundMath.mul(graphMarket.metrics.totalSupplyOnPool, graphMarket.reserveData.supplyPoolIndex),
+  totalPoolBorrowUSD: BigNumber.from(graphMarket.metrics.totalBorrowOnPool),
   p2pIndexCursor: BigNumber.from(graphMarket.p2pIndexCursor),
-  totalMorphoSupply: CompoundMath.mul(graphMarket.metrics.supplyBalanceInP2P, graphMarket.p2pData.p2pSupplyIndex).add(
-    CompoundMath.mul(graphMarket.metrics.supplyBalanceOnPool, graphMarket.reserveData.supplyPoolIndex)
-  ),
-  totalMorphoBorrow: CompoundMath.mul(graphMarket.metrics.borrowBalanceInP2P, graphMarket.p2pData.p2pBorrowIndex).add(
-    CompoundMath.mul(graphMarket.metrics.borrowBalanceOnPool, graphMarket.reserveData.borrowPoolIndex)
-  ),
+  morphoSupplyMarketSize: CompoundMath.mul(
+    graphMarket.metrics.supplyBalanceInP2P,
+    graphMarket.p2pData.p2pSupplyIndex
+  ).add(CompoundMath.mul(graphMarket.metrics.supplyBalanceOnPool, graphMarket.reserveData.supplyPoolIndex)),
+  morphoBorrowMarketSize: CompoundMath.mul(
+    graphMarket.metrics.borrowBalanceInP2P,
+    graphMarket.p2pData.p2pBorrowIndex
+  ).add(CompoundMath.mul(graphMarket.metrics.borrowBalanceOnPool, graphMarket.reserveData.borrowPoolIndex)),
```

Comment:
I'm just noting that we're using `poolBorrowIndex` in the contracts, it might be a bit misleading. And later in this PR there's `totalPoolBorrow` for instance